pyproject.toml

```diff
@@ -142,4 +142,4 @@ skip = [".venv", ".git", ".cache", ".tox"]
 
 [build-system]
 requires = ["poetry-core>=1.0.0", "poetry-dynamic-versioning"]
-build-backend = "poetry.core.masonry.api"
+build-backend = "poetry_dynamic_versioning.backend"
```

Comment:
This probably would have fixed the 0.0.0 version CI issue that we hit recently.
Thanks for the fix!